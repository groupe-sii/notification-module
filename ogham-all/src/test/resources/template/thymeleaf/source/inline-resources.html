<!DOCTYPE html>
<html xmlns:th="http://www.thymeleaf.org">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<title>Inline resources</title>
<meta name="viewport" content="width=device-width, initial-scale=1.0"/>

<link href="css/background-images.css" rel="stylesheet" />

</head>
<body>
	<div class="header">
		<img class="logo" src="images/h1.gif" />
	</div>
	<div class="footer">
		footer
		<img class="fb" src="images/fb.gif" data-inline-image="base64" />
	</div>
</body>
</html>