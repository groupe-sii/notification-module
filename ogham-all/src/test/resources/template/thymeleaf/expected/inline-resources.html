<!doctype html>
<html>
 <head> 
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> 
  <title>Inline resources</title> 
  <meta name="viewport" content="width=device-width, initial-scale=1.0"> 
 </head> 
 <body> 
  <div class="header" style="background: #ccc url(&quot;cid:1&quot;) repeat-x; height: 300px;"> 
   <img class="logo" src="cid:0"> 
  </div> 
  <div class="footer" style="background: url(&quot;data:image/jpeg;base64,/9j/4QAYRXhpZgAASUkqAAgAAAAAAAAAAAAAAP/sABFEdWNreQABAAQAAABLAAD/4QMsaHR0cDovL25zLmFkb2JlLmNvbS94YXAvMS4wLwA8P3hwYWNrZXQgYmVnaW49Iu+7vyIgaWQ9Ilc1TTBNcENlaGlIenJlU3pOVGN6a2M5ZCI/PiA8eDp4bXBtZXRhIHhtbG5zOng9ImFkb2JlOm5zOm1ldGEvIiB4OnhtcHRrPSJBZG9iZSBYTVAgQ29yZSA2LjAtYzAwMiA3OS4xNjQzNTIsIDIwMjAvMDEvMzAtMTU6NTA6MzggICAgICAgICI+IDxyZGY6UkRGIHhtbG5zOnJkZj0iaHR0cDovL3d3dy53My5vcmcvMTk5OS8wMi8yMi1yZGYtc3ludGF4LW5zIyI+IDxyZGY6RGVzY3JpcHRpb24gcmRmOmFib3V0PSIiIHhtbG5zOnhtcD0iaHR0cDovL25zLmFkb2JlLmNvbS94YXAvMS4wLyIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bXA6Q3JlYXRvclRvb2w9IkFkb2JlIFBob3Rvc2hvcCAyMS4xIChXaW5kb3dzKSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDo2NEY1ODY2RUE3MzcxMUVBOTc1MTg5REY3NUZBMUU1MCIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDo2NEY1ODY2RkE3MzcxMUVBOTc1MTg5REY3NUZBMUU1MCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOjY0RjU4NjZDQTczNzExRUE5NzUxODlERjc1RkExRTUwIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjY0RjU4NjZEQTczNzExRUE5NzUxODlERjc1RkExRTUwIi8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+/+4ADkFkb2JlAGTAAAAAAf/bAIQAAwICAgICAwICAwUDAwMFBQQDAwQFBgUFBQUFBggGBwcHBwYICAkKCgoJCAwMDAwMDA4ODg4OEBAQEBAQEBAQEAEDBAQGBgYMCAgMEg4MDhIUEBAQEBQREBAQEBARERAQEBAQEBEQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQEBAQ/8AAEQgBLAABAwERAAIRAQMRAf/EAFEAAQEBAAAAAAAAAAAAAAAAAAMCCQEBAQAAAAAAAAAAAAAAAAAAAAEQAQEBAQEAAAAAAAAAAAAAAAASEQFhEQEAAAAAAAAAAAAAAAAAAAAA/9oADAMBAAIRAxEAPwDVLeAkEAnVEaIjQRoIrig6AdeiIoB0oKgRQgqUHXoDoQVKDoBUA6UFQg6UHQCoRFAKlB16CKAe9VB7wEaCNBGiJpRFcBO8BIP/2Q==&quot;) no-repeat; height: 300px; ">
    footer 
   <img class="fb" src="data:image/gif;base64,R0lGODlhJgAmANUAAMs8QLMxNeNHS+pKTuFGSrw1OdtDR7YyNudJTbIwNLEwNO1MULAwNMg7P+1LT7AvM7s0ONhCRuxLT8Q5PelKTtxESLUyNutLT782OuVITLgzN99FSehJTbk0OMk7P9lCRsI4PMo7P9A+QrczN+RITNZBRcc6Pr83O701OcE3O9RARMY6PuZITN5ESOBGSr42OtpDR7kzN9dBRbo0OM09QcM4PNI/Q9M/Q8A3O9NARORHS91ESLQxNd5FSa8vM+5MUCH/C1hNUCBEYXRhWE1QPD94cGFja2V0IGJlZ2luPSLvu78iIGlkPSJXNU0wTXBDZWhpSHpyZVN6TlRjemtjOWQiPz4gPHg6eG1wbWV0YSB4bWxuczp4PSJhZG9iZTpuczptZXRhLyIgeDp4bXB0az0iQWRvYmUgWE1QIENvcmUgNS4wLWMwNjAgNjEuMTM0Nzc3LCAyMDEwLzAyLzEyLTE3OjMyOjAwICAgICAgICAiPiA8cmRmOlJERiB4bWxuczpyZGY9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkvMDIvMjItcmRmLXN5bnRheC1ucyMiPiA8cmRmOkRlc2NyaXB0aW9uIHJkZjphYm91dD0iIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtbG5zOnhtcE1NPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvbW0vIiB4bWxuczpzdFJlZj0iaHR0cDovL25zLmFkb2JlLmNvbS94YXAvMS4wL3NUeXBlL1Jlc291cmNlUmVmIyIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDo1QjcwQTA5RkJFMTUxMUUyQjUwQ0YzRjY0Nzg2NURGMCIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDo1QjcwQTBBMEJFMTUxMUUyQjUwQ0YzRjY0Nzg2NURGMCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOjVCNzBBMDlEQkUxNTExRTJCNTBDRjNGNjQ3ODY1REYwIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjVCNzBBMDlFQkUxNTExRTJCNTBDRjNGNjQ3ODY1REYwIi8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+Af/+/fz7+vn49/b19PPy8fDv7u3s6+rp6Ofm5eTj4uHg397d3Nva2djX1tXU09LR0M/OzczLysnIx8bFxMPCwcC/vr28u7q5uLe2tbSzsrGwr66trKuqqainpqWko6KhoJ+enZybmpmYl5aVlJOSkZCPjo2Mi4qJiIeGhYSDgoGAf359fHt6eXh3dnV0c3JxcG9ubWxramloZ2ZlZGNiYWBfXl1cW1pZWFdWVVRTUlFQT05NTEtKSUhHRkVEQ0JBQD8+PTw7Ojk4NzY1NDMyMTAvLi0sKyopKCcmJSQjIiEgHx4dHBsaGRgXFhUUExIREA8ODQwLCgkIBwYFBAMCAQAAIfkEAAAAAAAsAAAAACYAJgAABv/An3BILP4WBIKL4DA6n85BpJbwWa8KUIkC7Q4Rpqt4bJ1kvM4LQCFOFCaNxqQQEDM8A/SQoLkmGgZNRQsVIXVWBxt6FWw+Dx5cXgNrVgwfXi0MVhoCekMZM1YPMFA6jRh5nkMXKZUETg4dViiCqkMLOFYjqUQiiJFdJEm8QwN9PjRFA4cuXjRVVgZOAg8+CQhEAGVeJWPSTg1WJkMSdQ9nXQVXHQWKTghsCqkGViBoh99dK1YyQh5WEe5Z6eSFno8JQtT54CDQB8EuF6qNEFIlQBeDZCp0OSZhgJUCF8lYYdGllQ8ECKzgCEkmgZcJAzlYwdBlQBJoHwg8fALCConGC1Yg6Dm0EwoGK9gsWBs6UM8BHwwW/DjR1AtRNCl9CP1Bw4qNhkWddPMBQAiBj2DRUPWhUUiMaFarQtlgxYJUISqC3n1ytcsLKyKISHjq40aXvk8iWAlA7IdiqD1sGSHQSIWTMD54kJD8hXCNJw7++gjQgjMBpVolQKHw1pGIWl4W3NDk4wC2mkd1BfQCQ9ZHhmgchBBjARCwHwN2hCBsZYVqVRsUjjnAXAyEtpw/oBA5ZkY/zkVY5GhQQCGEAg1soAPPvksQADs="> 
  </div>  
 </body>
</html>